yarn.lock

```diff
@@ -2706,29 +2706,7 @@
   resolved "https://registry.yarnpkg.com/@juggle/resize-observer/-/resize-observer-3.4.0.tgz#08d6c5e20cf7e4cc02fd181c4b0c225cd31dbb60"
   integrity sha512-dfLbk+PwWvFzSxwk3n5ySL0hfBog779o8h68wK/7/APo/7cgyWp5jcXockbxdk5kFRkbeXWm4Fbi9FrdN381sA==
 
-"@kaizen/brand-moment@^1.13.8":
-  version "1.13.8"
-  resolved "https://registry.yarnpkg.com/@kaizen/brand-moment/-/brand-moment-1.13.8.tgz#662a68f17d9237cbf35dbfddf4c49227aada72a4"
-  integrity sha512-+LCh6BiDnAOpu4Q5cJ5t+pUl4wlDz6otmrhdNdmmzYg064UuCWFAKmZqD7HhTHXTHEjgR66yhoQefu2bu9YotQ==
-  dependencies:
-    "@kaizen/button" "^3.0.4"
-    "@kaizen/component-base" "^1.1.7"
-    "@kaizen/component-library" "^16.9.1"
-    "@kaizen/draft-illustration" "^7.0.0"
-    "@kaizen/hosted-assets" "^2.0.3"
-    "@kaizen/responsive" "^1.8.10"
-    "@kaizen/typography" "^2.3.11"
-    classnames "^2.3.2"
-
-"@kaizen/brand@^1.5.8":
-  version "1.5.8"
-  resolved "https://registry.yarnpkg.com/@kaizen/brand/-/brand-1.5.8.tgz#934d90e710c3f82b173e06c58111215c3bca40d6"
-  integrity sha512-XGbgwhyBUNjF7/GONBHKpEmVar6TfiTYu7sU/tLzcjQroasa9bf0NBf4Lp0MhUHSXXHShRX1tdRW038zfDtCFQ==
-  dependencies:
-    "@kaizen/component-base" "^1.1.7"
-    "@kaizen/hosted-assets" "^2.0.3"
-
-"@kaizen/button@^3.0.4", "@kaizen/button@^3.0.5":
+"@kaizen/button@^3.0.4":
```

Comment:
If all components are migrated before this is merged, we can check the yarn.lock no longer has any legacy deps.